README.md

```diff
@@ -253,7 +255,8 @@ python -m ai_feedback -h
 
 #### Evaluate cnn_example test using openAI model 
 ```bash
-python -m ai_feedback --prompt code_lines --scope code --submission test_submissions/cnn_example/cnn_submission --solution test_submissions/cnn_example/cnn_solution.py --model openai --output stdout
+python -m ai_feedback --prompt code_lines --scope code --submission test_submissions/cnn_example/cnn_submission --solution test_submissions/cnn_example/cnn_solution.py --model openai --system_prompt ai_feedback/data/prompts/system/student_test_feedback.md
```

Comment:
You don't need to provide the `--system_prompt` argument for this example because it's a default value; instead, create a different example that illustrates passing a custom system prompt, similar to the example below